lib/Controller/FolderApiController.php

```diff
@@ -94,7 +94,7 @@ public function create(string $name)
      */
     public function delete(?int $folderId)
     {
-        if (empty($folderId)) {
+        if (is_null($folderId)) {
```

Comment:
Well, there wouldn't be a 0 to remove. And null now short-circuits

Reply:
Therefore STATUS_NOT_FOUND (404) would be returned instead of STATUS_BAD_REQUEST (400)?
If the behaviour for the API remains the same, apart from that, then it's ok for me.